addons/base/common/package/index.js

```diff
@@ -34,7 +34,15 @@ module.exports = function resolvePackage(setup, { appName, command }) {
       "test": "jest --runInBand --detectOpenHandles --passWithNoTests",
       "test:watch": "jest -u --runInBand --verbose --watch --detectOpenHandles --passWithNoTests",
       "test:coverage": "jest -u --coverage --verbose --runInBand --detectOpenHandles --passWithNoTests",
-    }
+    },
+    husky: {
+        "hooks": {
+          "pre-commit": "lint-staged"
+        }
+      },
+      "lint-staged": {
+        "*.js": ["yarn lint:fix", "git add"]
```

Comment:
Only for `*.js` files? I think we'll need it for `jsx`, `ts`, `tsx` too. I'll update it later if you want